changelog/9886.deprecation.rst

```diff
@@ -0,0 +1 @@
+The functionality for running tests written for ``nose`` has been officially deprecated.
```

Comment:
This should be a bit more verbose: It should point out that this also includes `setup` and `teardown` methods (since users don't seem to be aware of that being a nose thing), and it should have a link to the detailed deprecation docs.

Reply:
Indeed, done. 👍